### ISSUE TITLE
Add other apps to Application managment

### ISSUE THREAD

**Issue by Mcpower03** [egee-irl/distro-delves#3]
## Add, Remove, or Update?
 Add a less common application or two to the app installs, such as natron or qwinff as well as a wine program and something that needs to be installed from the internet, such as davinci resolve or unity 3d

## Which Segment?
Add to application management 

## Rationale?
This would give a better scope of the repos of a given distro and what kind of more obscure support is available. Also how one click some programs that are not in the repos are

**Comment by egee-irl** [comment]
These are great choices, I'll see if I can fit them into the script

**Comment by egee-irl** [comment]
I've reconsidered this task and I've decided to remove the "mundane" apps from the install list (Audacity, Wine, etc) and leave the specialized or "enterprise-y" apps, such as Docker, TeamViewer, Zoom, etc.

I think Natron, Davinci Resolve, and possibly Unity 3D would make excellent additions as they are big, enterprise-grade, and troublesome to get running on some distros.

Thanks for the recommendations!
